docs/ref/models/querysets.txt

```diff
@@ -2445,6 +2432,34 @@ more overall work (one query for the existence check plus an extra one to later
 retrieve the results) than using ``bool(some_queryset)``, which retrieves the
 results and then checks if any were returned.
 
+``contains()``
+~~~~~~~~~~~~~~
+
```

Comment:
`versionadded` is missing.

Reply:
versionadded is yet to be decided, no?

Reply:
There is nothing to decide, the current branch under development is Django 3.2.